src/state/cache/thread-mutes.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+type StateContext = Record<string, boolean>
+type SetStateContext = (uri: string, value: boolean) => void
+
+const stateContext = React.createContext<StateContext>({})
+const setStateContext = React.createContext<SetStateContext>(
+  (_: string) => false,
+)
+
+export function Provider({children}: React.PropsWithChildren<{}>) {
```

Comment:
This provider is currently rendered _above_ the part that we key on `did`. Should it be moved to be below now? I assume the setting is per-account rather than per-device now?